models/event.js

```diff
@@ -0,0 +1,13 @@
+const { connector } = require("./databaseUtil");
+
+const eventSchema = new connector.Schema({
+    eventName: { type: String, required: true },
+    date: { type: Date, required: true },
+    infra: { type: String, required: true },
```

Comment:
infra will be of type:Infrastructure and add attendance attribute which be be of type:[Attendance]

Reply:
put the necessary imports as well